docker/picard/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM java:openjdk-8-jre
+FROM openjdk:8-jre
```

Comment:
Can you explain the reason for this change?

Reply:
Sure, according to https://hub.docker.com/_/java/ , the `java` base image is officially deprecated in favor of the `openjdk` image